share/locale/en.json

```diff
@@ -1,165 +1,165 @@
+    "gui_quit_warning": "Quitting now means files underway won't finish.",
     "gui_quit_warning_quit": "Quit",
     "gui_quit_warning_dont_quit": "Cancel",
-    "error_rate_limit": "An attacker might be trying to guess your address. To prevent this, OnionShare has automatically stopped the server. To share the files you must start it again and share the new address.",
-    "zip_progress_bar_format": "Compressing files: %p%",
-    "error_stealth_not_supported": "To create stealth onion services, you need at least Tor 0.2.9.1-alpha (or Tor Browser 6.5) and at least python3-stem 1.5.0.",
+    "error_rate_limit": "Too many guesses means someone could be trying to work out the address of your files. OnionSharing has stopped. Start it again and send the new address to share them.",
+    "zip_progress_bar_format": "Making files smaller: %p%",
+    "error_stealth_not_supported": "To create onion services, you need at least Tor 0.2.9.1-alpha (or Tor Browser 6.5) and at least python3-stem 1.5.0.",
     "error_ephemeral_not_supported": "OnionShare requires at least Tor 0.2.7.1 and at least python3-stem 1.4.0.",
     "gui_settings_window_title": "Settings",
     "gui_settings_stealth_label": "Stealth (advanced)",
-    "gui_settings_stealth_option": "Create stealth onion services",
-    "gui_settings_stealth_option_details": "This makes OnionShare more secure, but also more difficult for the recipient to connect to.<br><a href=\"https://github.com/micahflee/onionshare/wiki/Stealth-Onion-Services\">More information</a>.",
-    "gui_settings_stealth_hidservauth_string": "You have saved the private key for reuse, so your HidServAuth string is also reused.\nClick below to copy the HidServAuth.",
-    "gui_settings_autoupdate_label": "Check for upgrades",
-    "gui_settings_autoupdate_option": "Notify me when upgrades are available",
+    "gui_settings_stealth_option": "Create onion services that are stealthy",
+    "gui_settings_stealth_option_details": "This is more secure, but also more difficult to connect to.<br><a href=\"https://github.com/micahflee/onionshare/wiki/Stealth-Onion-Services\">More info</a>.",
+    "gui_settings_stealth_hidservauth_string": "Having saved your private key for reuse, means you can now\nclick to copy your HidServAuth.",
+    "gui_settings_autoupdate_label": "Check for new versions",
+    "gui_settings_autoupdate_option": "Notify me when a new version is available",
     "gui_settings_autoupdate_timestamp": "Last checked: {}",
     "gui_settings_autoupdate_timestamp_never": "Never",
-    "gui_settings_autoupdate_check_button": "Check For Upgrades",
-    "gui_settings_sharing_label": "Sharing options",
-    "gui_settings_close_after_first_download_option": "Stop sharing after first download",
+    "gui_settings_autoupdate_check_button": "Check For New Versions",
+    "gui_settings_sharing_label": "OnionSharing options",
+    "gui_settings_close_after_first_download_option": "Stop after first OnionSharing",
```

Comment:
I like just "Sharing options" and "Stop after first download" because of how the settings dialog is broken up now

Reply:
I am pondering "delivery", which seems to work for parcels of multiple packets.